Editor/Linux/Install.py

```diff
@@ -1,32 +1,61 @@
 #!/usr/bin/python
 
 import os
+import time 
 
-os.system('clear')
+os.system('clear') # Clear the console to make things easier to read.
 
 answer_installModuleDependencies = input("This installer has some Python dependencies, would you like to install them? [Yes/No]" + '\n')
 
 if answer_installModuleDependencies == "Yes":
     print("Installing required modules.")
     os.system('pip install pillow ttkbootstrap distro')
 elif answer_installModuleDependencies == "No": 
-    print("Skipping module dependencies!" + '\n' + "The installer might not run.")
-elif answer_installModuleDependencies == "Yes":
+    print("Skipping module dependencies!")
+    print("The installer might not run.")
+    time.sleep(1)
+elif answer_installModuleDependencies == "yes":
```

Comment:
there is a better way to do this, that removes the code duplication:
```if answer_installModuleDependencies.lower() == "yes":
...
elif answer_installModuleDependencies.lower() == "no":
...
```

Reply:
Oh, thanks! I didn't know about `.lower`, I'll make sure to apply that.